src/Microsoft.DocAsCode.Build.SchemaDriven/Models/OverwriteModelType.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.SchemaDriven
+{
+    public enum OverwriteModelType
+    {
+        OverwriteDocument,
```

Comment:
naming: `Traditional`?

Reply:
classic vs modern?

---
In reply to: [165293683](https://github.com/dotnet/docfx/pull/2398#discussion_r165293683) [](ancestors = 165293683)

Reply:
Classic?